app/models/concerns/deleted_user_relation_fallback.rb

```diff
@@ -0,0 +1,9 @@
+module DeletedUserRelationFallback
+  extend ActiveSupport::Concern
+
+  included do
+    def user
+      super || User.deleted.find_by(id: user_id)
+    end
+  end
+end
```

Comment:
Had a look at the paranoia README to see if there's a more general way to deal with this, one option would be to specify a scope on the association:

> If you want to include associated soft-deleted objects, you can (un)scope the association:
> 
> ``` ruby
> class Person < ActiveRecord::Base
>   belongs_to :group, -> { with_deleted }
> end
> 
> Person.includes(:group).all
> ```

Should be more reliable, and also avoids running another DB query for deleted users.